onnxruntime/contrib_ops/cuda/math/isfinite_impl.h

```diff
@@ -0,0 +1,18 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
noticed that contrib isfinite deviates from the pattern of x.h, x.cc, x_impl.h, x_impl.cu
why is that?